packages/application-shell/src/components/quick-access/butler/butler.tsx

```diff
@@ -12,7 +12,7 @@ import { css, keyframes, ClassNames } from '@emotion/react';
 import Fuse from 'fuse.js';
 import last from 'lodash/last';
 import { FormattedMessage, useIntl } from 'react-intl';
-import { customProperties } from '@commercetools-uikit/design-system';
+import { designTokens } from '@commercetools-uikit/design-system';
```

Comment:
nit: for consistency
```suggestion
import { designTokens as uiKitDesignTokens } from '@commercetools-uikit/design-system';
```

Reply:
Updated here: 83560abf88251355859415b1b7577d32f0ed8b41